src/applications/caregivers/tests/unit/components/FormPages/FacilityConfirmation.unit.spec.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import userEvent from '@testing-library/user-event';
+import { mockLightHouseFacilitiesResponseWithTransformedAddresses } from '../../../mocks/responses';
+import FacilityConfirmation from '../../../../components/FormPages/FacilityConfirmation';
+
+describe('CG <FacilityConfirmation>', () => {
+  const facilities =
+    mockLightHouseFacilitiesResponseWithTransformedAddresses.data;
+  const selectedFacility = facilities[0];
+
+  const goBack = sinon.spy();
+  const goForward = sinon.spy();
+
+  const defaultProps = {
+    data: {
+      'view:plannedClinic': {
+        veteranSelected: facilities[0],
+      },
+    },
+    goBack,
+    goForward,
+  };
+
+  const subject = (props = defaultProps) => {
+    const { getByText, getByRole } = render(
+      <FacilityConfirmation {...props} />,
+    );
+    const selectedFacilityAddress =
+      selectedFacility.attributes.address.physical;
+
+    const selectors = () => ({
```

Comment:
Should we really be selecting objects by text? I dislike this method, mainly because I don't want a test to break just because some piece of content changes.

Reply:
Wouldn't we _want_ tests to break if a piece of content changes? If we have a typo in the component, this would catch it if it is wrong. At the very least, having to have the text in both places means we'd have to type the wrong thing twice. If we just test that a header exists, it feels much easier to have the wrong thing in the header.

Reply:
Personally, I don't like the idea of tests breaking for content changes that aren't a part of some sort of conditional display. If we have an if/else display of some message, sure, lets do a test for that But otherwise, it seems a bit silly.

Reply:
Yeah, that makes sense too. I like the use of the getByLabel on this at least, since we care about that for accessibility. I ended up pulling out the text part and instead used validate what text is rendered for the labels from the selector. I think that makes it a little clearer anyways, even if it is maybe overkill/unnecessary for simple text content.